load_balancers.go

```diff
@@ -104,10 +103,6 @@ func (l LoadBalancer) AsRequest() *LoadBalancerRequest {
 		r.Region = l.Region.Slug
 	}
 
-	if l.HTTPIdleTimeoutSeconds == nil {
```

Comment:
setting this here https://github.com/digitalocean/godo/pull/580/files#diff-c9b4c15afcc0c5f0ead492bd8e12abb791b72cdd4985919a15f04b7708629281R87 should be sufficient

Reply:
@StephenVarela is the diff link supposed to reference a specific line?

Reply:
Hrmm thats weird sorry I meant to point to line 88 in this file (load_balancers.go)  https://github.com/digitalocean/godo/blob/main/load_balancers.go#L88

Reply:
I just understood your wording "setting this here ... should be sufficient" to mean the link was pointing elsewhere since the comment is on removed code.